lib/active_fedora/file.rb

```diff
@@ -139,6 +140,19 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      mime_type[/url=['"](.+?)['"]/i, 1]
+    end
+
+    def external_url=(external_file_url)
+      self.mime_type = "message/external-body; access-type=URL; URL=\"#{external_file_url}\""
```

Comment:
I updated the mime_type in Fedora directly via SPARQL. Using `access_type=URL` with `URL=http://[etc]` worked fine. When I clicked "Download content" in the Fedora UI, it redirected to the content. Note, it actually didn't "download" it, it presented the image in the window, which is a true redirect.

When I used either `URL=file:///` to a local path, or @escowles `local-file` option, neither worked. I clicked on "Download content" and nothing happened. I didn't look at logs, though. But my point was that maybe we should just support a basic HTTP protocol first, and work on others later.